_docs-2/configuration/overview.md

```diff
@@ -59,6 +59,17 @@ They can also be set using {% jlink org.apache.accumulo.core.client.admin.Instan
 client.instanceOperations().setProperty("table.durability", "flush");
 ```
 
+The java api also supports adding, modifying and removing multiple properties in a single operation:
+
+```java
+client.instanceOperations().modifyProperties(properties -> {
+  properties.remove(Property.TABLE_FILE_MAX.getKey());
```

Comment:
`org.apache.accumulo.core.conf.Property` is not public API, and should not be used in documentation as an example. It should be replaced with the actual String key for the property, which is the user-facing "public API" for configuration.

Reply:
Fixed in 4dbd6b5f95a406d790fcbbd5182ed9fbcdef3f75